index.html

```diff
@@ -17,102 +17,120 @@
       <a href="#" class="logo"><h2>John Doe</h2></a>
       <nav>
         <i class="fa-solid fa-bars fa-2x"></i>
+        <ul>
+          <li>Portoflio</li>
+          <li>About</li>
+          <li>Contact</li>
+          <li><i class="fa-regular fa-square-caret-down"></i></li>
+        </ul>
       </nav>
     </header>
-    <section class="headline">
-      <hgroup>
-        <h1 class="head-main-title">Hey There. I'm Alex</h1>
-        <h3 class="head-main-sub-title">I’am a Software Developer</h3>
-      </hgroup>
-      <p class="head-main-description">
-        I can help you build a product , 
-        feature or website Look through some of my work and experience! 
-        If you like what you see and have a project you need coded, 
-        don’t hestiate to contact me.
-      </p>
+    <section class="headline-wrap">
+      <section class="headline">
+        <hgroup>
+          <h1 class="head-main-title">Hey There. I'm Getayawkal</h1>
+          <h3 class="head-main-sub-title">I’am a Software Developer</h3>
+        </hgroup>
+        <p class="head-main-description">
+          I can help you build a product , 
+          feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.
+        </p>
+      </section>
+      <ul class="social">
+          <li><a href=""><i class="fa-brands fa-github"></i></a></li>
+          <li><a href=""><i class="fa-brands fa-linkedin-in"></i></a></li>
+          <li><a href=""><i class="fa-regular fa-hand-peace"></i></a></li>
+          <li><a href=""><i class="fa-brands fa-twitter"></i></a></li>
+          <li><a href=""><i class="fa-solid fa-m"></i></a></li>
```

Comment:
- [ ] [ OPTIONAL ] ![listst1'](https://user-images.githubusercontent.com/87394385/213766249-db24d144-74b2-400d-ac6c-8a6e7a646d87.JPG)

-   To make your interface look better, please consider removing this default style. You can achieve that by adding this CSS rule : `.social li { list-style: none; }`